.github/workflows/ci.yml

```diff
@@ -214,42 +214,4 @@ jobs:
           go-version: 1.23
       - name: Run e2e TestBTCStakingPreApprovalTestSuite
         run: |
-          make test-e2e-cache-btc-staking-pre-approval
-
-  # Disable this test because v1 upgrade does not cover integration features
-  # TODO: Reenable this test after having v2 upgrade
-  e2e-run-upgrade-v1:
-    if: false
-    needs: [e2e-docker-build-babylon, e2e-docker-build-e2e-init-chain]
-    runs-on: ubuntu-22.04
-    steps:
-      - name: Checkout repository
-        uses: actions/checkout@v4
-      - name: Download babylon artifact
-        uses: actions/download-artifact@v4
-        with:
-          name: babylond-${{ github.sha }}
-          path: /tmp
-      - name: Download init-chain artifact
-        uses: actions/download-artifact@v4
-        with:
-          name: init-chain
-          path: /tmp
-      - name: Docker load babylond
-        run: |
-          docker load < /tmp/docker-babylond.tar.gz
-
-      - name: Docker load init chain
-        run: |
-          docker load < /tmp/docker-init-chain.tar.gz
-
-      - name: Cache Go
-        uses: actions/setup-go@v5
-        with:
-          go-version: 1.23
-      - name: Run e2e TestSoftwareUpgradeV1TestnetTestSuite
-        run: |
-          sudo make test-e2e-cache-upgrade-v1
-      - name: Dump docker logs on failure
-        if: failure()
-        uses: jwalton/gh-docker-logs@v2
+          make test-e2e-cache-btc-staking-pre-approval
```

Comment:
we should have this e2e v2 upgrade test on main, right?

Reply:
I had it but it was failing the test consistently due to phase 3 modules already being wired up so I removed it not to confuse other PRs.

Reply:
got it, makes sense